### ISSUE TITLE
Number column names fail when using fromCSV

### ISSUE THREAD

**Issue by wenyichen** [data-forge/data-forge-ts#97]
When using numbers as column names, the fromCSV function throws an exception `columnName.toLowerCase is not a function`

```js
const csvDataFrame = fromCSV(`"Column Name","123"\ndata1,data2`, {
    dynamicTyping: true,
});
```

**Comment by ashleydavis** [comment]
Thanks for reporting the issue. I'll try and have a look during the week.

**Comment by wenyichen** [comment]
Did a little more research into this, this only happens with `dynamicTyping: true` and only when the header doesn't have spaces before the number column name. Seems that the PapaParse library, given `"c1", "123"
abc,def` produces the expected output for the header with 123 as string, but given `"c1","123"
abc,def` changes the 123 in the column header to number type

**Comment by ashleydavis** [comment]
Thanks again for reporting this.  A fix is now available in version 1.8.16.
